app/furniture/marketplace/delivery_area.rb

```diff
@@ -22,7 +22,15 @@ def discardable?
     end
 
     def destroyable?
-      persisted? && discarded? && orders.empty?
+      persisted? && discarded? && unordered?
+    end
+
+    def ordered?
+      orders.present?
+    end
+
+    def unordered?
+      !ordered?
```

Comment:
I feel like it will be so easy to interpret these methods for sorting, although I understand this type of grammar is popular in ruby/Rails. Because of the potential confusion could an alternative like `has_orders` work here?

Reply:
I was trippin' over that myself, what about something like `with_orders?` and `without_orders?`Or I could just keep doing `ordes.present?` since I don't think it's the end of the world and adding in the predicate method could be confusing.

Reply:
I like `with_orders?`